### ISSUE TITLE
[Design] Claims Status: Begin Iterating on Early Design Concepts Post Usability #1 Synthesis

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#10838]
## Issue Description
Preparing for the design intent review with some initial design concepts.  Make sure to pull in accessibility!

Per Kevin (smart man), we really need to spend more time talking to stakeholders and have more of the tech discovery done before we can move into a real design intent.

## Acceptance Criteria
- [ ] Ready to pull in VSP and Accessibility into initial discussions

## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by kevinstachura** [comment]
We have begun iterating on early design concepts via our CST ideation sessions.

**Comment by kevinstachura** [comment]
Our initial concepts can be found here: https://vsateams.invisionapp.com/share/SUYB3J9FZ2Y

**Comment by luke-gcio** [comment]
Initial design concept complete, will work on another story for next round.
